tests/unit/dogstatsd/test_statsd.py

```diff
@@ -252,6 +252,95 @@ def test_accessing_socket_multiple_times_returns_same_socket(self):
         t.assert_not_equal(FakeSocket(), dogpound.get_socket())
 
 
+class TestDogStatsdConstantTags(object):
```

Comment:
I don't think we need to duplicate all tests to check the constant tag feature, as they all use the same codepath (i.e. `_report` method). I would just add, to the existing class test, a single test like:
```python
def test_constant_tags(self):
    self.statsd.constant_tags=...
    self.statsd.gauge('gauge', 123.4)
    assert self.recv() == 'gauge:123.4|g|#bar:baz,foo'
```

What do you think ?